runtests.py

```diff
@@ -40,7 +40,7 @@
     STATIC_URL='/static/',
     ROOT_URLCONF='djedi.tests.urls',
 
-    LANGUAGE_CODE='sv-se',
+    LANGUAGE_CODE='sv',
```

Comment:
Django>=3.0,<3.0.3 does not allow sublanguages even if the base language is present.

https://code.djangoproject.com/ticket/31141

Reply:
This might be hard for users upgrading to django 3, since all their djedi nodes might have namespaces containing sublanguages, eg `sv-se`.

Maybe as a separate PR we should add a utility management command that *fixes* namespaces and remove the sublanguage-part, if present? Or at least document this django change, and that it will affect finding proper nodes by language if upgrading to django 3.